tutor/src/screens/gradebook/task-result-cell.js

```diff
@@ -1,4 +1,5 @@
 import { React, styled, useObserver } from 'vendor';
+import { observer } from 'mobx-react';
```

Comment:
observer is exported by `vendor` - it's in tutor/src/vendor and should export pretty much everything